app/src/main/java/com/example/concordia_campus_guide/Activities/PathsActivity.java

```diff
@@ -0,0 +1,139 @@
+package com.example.concordia_campus_guide.Activities;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+import androidx.lifecycle.ViewModelProviders;
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.view.View;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+import com.example.concordia_campus_guide.Adapters.DirectionWrapper;
+import com.example.concordia_campus_guide.Fragments.LocationFragment.LocationFragment;
+import com.example.concordia_campus_guide.Fragments.PathInfoCardFragment.PathInfoCardFragment;
+import com.example.concordia_campus_guide.GoogleMapsServicesTools.GoogleMapsServicesModels.DirectionsResult;
+import com.example.concordia_campus_guide.GoogleMapsServicesTools.GoogleMapsServicesModels.DirectionsStep;
+import com.example.concordia_campus_guide.Models.Place;
+import com.example.concordia_campus_guide.Models.RoomModel;
+import com.example.concordia_campus_guide.Models.WalkingPoint;
+import com.example.concordia_campus_guide.R;
+import com.google.android.material.bottomsheet.BottomSheetBehavior;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PathsActivity extends AppCompatActivity {
+    PathsViewModel mViewModel;
+    LocationFragment locationFragment;
+    TextView fromTextView;
+    TextView toTextView;
+    ImageButton backButton;
+    PathInfoCardFragment pathInfoCardFragment;
+    FragmentTransaction fragmentTransaction;
```

Comment:
Don't forget to add the access modifiers.

Reply:
Thank you